server/proxy/enable.go

```diff
@@ -51,13 +61,23 @@ func (p *Proxy) DisableApp(cc *apps.Context, app *apps.App) (md.MD, error) {
 		return "no change.", nil
 	}
 
+	cc, err = p.CleanUserCallContext(cc.ActingUserID, cc)
+	if err != nil {
+		return "", err
+	}
+
 	app.Disabled = true
-	resp := p.Call("", "", &apps.CallRequest{
-		Call:    *app.OnDisable,
-		Context: cc,
-	})
-	if resp.Type == apps.CallResponseTypeError {
-		p.mm.Log.Warn("OnDisable failed, app disabled anyway", "err", resp.Error(), "app_id", app.AppID)
+
```

Comment:
Is the comment at https://github.com/mattermost/mattermost-plugin-apps/blob/951f54b169dcda31aca45f71f189606036a5027e/apps/manifest.go#L62 still valid?

Reply:
The functions exist but they are not "supported" as those functions are never called.